apps/web/app/root.tsx

```diff
@@ -261,6 +262,11 @@ function Document({
             defer
           ></script>
         )}
+        <script
```

Comment:
worth bundling this at all? ill have to dig into it more to understand

is this just compat ala does it only add browser support?